stdlib/obj.ml

```diff
@@ -18,8 +18,9 @@ type t
 external repr : 'a -> t = "%identity"
 external obj : t -> 'a = "%identity"
 external magic : 'a -> 'b = "%identity"
-external is_block : t -> bool = "caml_obj_is_block"
 external is_int : t -> bool = "%obj_is_int"
+let is_block a = is_int a |> not
+  [@@ocaml.inline]
```

Comment:
Cf http://caml.inria.fr/mantis/view.php?id=7126 .  I think we should mark those functions for which we really don't want to loose inlining.  It's not noise, since it implements useful proper non-regression assertions (against changes in the stdlib code base or in the optimizer).